ripped from Debian

http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=493659
patch by nbreen@ofb.net (Nicholas Breen)

--- nfs-utils-1.1.3/utils/mount/nfs.man
+++ nfs-utils-1.1.3/utils/mount/nfs.man
@@ -799,7 +799,7 @@
 and server load.
 .P
 However, UDP can be quite effective in specialized settings where  
-the network’s MTU is large relative to NFS’s data transfer size (such  
+the network's MTU is large relative to NFS's data transfer size (such  
 as network environments that enable jumbo Ethernet frames).  In such  
 environments, trimming the 
 .B rsize 
